### PR TITLE
What's New in .NET Framework 4.8

### PR DESCRIPTION
**Pull request by rpetrusha**
## What's New in .NET Framework 4.8

What's New in Accessibility will be submitted in a separate PR.

### CODE REVIEW

**Comment by rpetrusha** [comment]
@NKarnam15 and @vivmishra, could you review this PR, please?

File: docs/framework/whats-new/index.md [hunk 1]
```diff
@@ -29,29 +30,162 @@ This article summarizes key new features and improvements in the following versi
...
+
+**NGEN improvements**. The runtime has improved its memory management for [Native Image Generator](../tools/ngen-exe-native-image-generator.md) (NGEN) images so that data mapped from NGEN images are not memory-resident. This reduces the surface area available to attacks that attempt to execute arbitrary code by modifying memory that will be executed.
+
+**Antimalware scanning for all assemblies**. In previous versions of the .NET Framework, the runtime scans all assemblies loaded from disk using either Windows Defender or third-party antimalware software. However, assemblies loaded from other sources, such as by the <xref:System.Reflection.Assembly.Load(System.Byte[])?displayProperty=nameWithType> method, are not scanned and can potentially contrain undetected malware. Starting with .NET Framework 4.8 running on Windows 10, the runtime triggers a scan by antimalware solutions that implement the [Antimalware Scan Interface (AMSI)](/windows/desktop/AMSI/antimalware-scan-interface-portal).
```
Comment (mairaw) [inline docs/framework/whats-new/index.md]:
```suggestion
**Antimalware scanning for all assemblies**. In previous versions of the .NET Framework, the runtime scans all assemblies loaded from disk using either Windows Defender or third-party antimalware software. However, assemblies loaded from other sources, such as by the <xref:System.Reflection.Assembly.Load(System.Byte[])?displayProperty=nameWithType> method, are not scanned and can potentially contain undetected malware. Starting with .NET Framework 4.8 running on Windows 10, the runtime triggers a scan by antimalware solutions that implement the [Antimalware Scan Interface (AMSI)](/windows/desktop/AMSI/antimalware-scan-interface-portal).  
```

**Review by mairaw — APPROVED** [review]
LGTM. Left some comments for you to consider.

File: docs/framework/whats-new/index.md [hunk 2]
```diff
@@ -29,29 +30,162 @@ This article summarizes key new features and improvements in the following versi
...
+- [Core](#core48)
+- [Windows Communication Foundation (WCF)](#wcf48)
+- [Windows Presentation Foundation (WPF)](#wpf48)
+- [Common language runtime](#clr48) 
+
+Improved accessibility, which allows an application to provide an appropriate experience for users of Assistive Technology, continues to be a major focus of .NET Framework 4.8. For information on accessibility improvements in .NET Framework 4.8, see [What's new in accessibility in the .NET Framework](whats-new-in-accessibility.md).
+
+<a name="core48" />
+
+#### Core
```
Comment (rpetrusha) [inline docs/framework/whats-new/index.md]:
```suggestion
#### Base classes
```

File: docs/framework/whats-new/index.md [hunk 3]
```diff
@@ -29,29 +30,162 @@ This article summarizes key new features and improvements in the following versi
...
+By default in applications that target .NET Framework 4.8, the following managed cryptography classes no longer throw a <xref:System.Security.Cryptography.CryptographicException> in this case:
+
+- <xref:System.Security.Cryptography.MD5Cng>
+- <xref:System.Security.Cryptography.MD5CryptoServiceProvider>
+- <xref:System.Security.Cryptography.RC2CryptoServiceProvider>
+- <xref:System.Security.Cryptography.RijndaelManaged>
+- <xref:System.Security.Cryptography.RIPEMD160Managed>
+- <xref:System.Security.Cryptography.SHA256Managed> 
+
+Instead, these classes redirect cryptographic operations to a system cryptography library. This change effectively removes a potentially confusing difference between developer environments and production environments and makes native components and managed components operate under the same cryptographic policy. Applications that depend on these exceptions can restore the previous behavior by setting the AppContext switch `Switch.System.Security.Cryptography.UseLegacyFipsThrow` to `true`. For more information, see [](..//migration-guide/retargeting/4.7.2-4.8.md#managed-cryptography-classes-do-not-throw-a-cryptographyexception-in-fips-mode).
```
[the remark this answers is not in the document]
Comment (rpetrusha) [reply docs/framework/whats-new/index.md]: The link should resolve once the compat documentation is merged.

**Pull request merged**